src/modules/routes.ts

```diff
@@ -3,8 +3,7 @@ import type http from 'node:http'
 import type { CommonLogger } from '@lokalise/node-core'
 import type { RouteOptions } from 'fastify'
 import type { ZodTypeProvider } from 'fastify-type-provider-zod'
-
-import { getUserRoutes } from './users/index.js'
+import { getUserRoutes } from './users/routes/userRoutes.js'
```

Comment:
IDE keeps recommending to simplify this into just `'./users/`, which doesn't work at a runtime in ESM